here,  
null Hyp (Ho) = non-stationary  
alternate Hyp (Ha) = stationary  

In [71]:
import os
import pandas as pd
from statsmodels.tsa.stattools import adfuller
pwd = os.getcwd()
file_path = pwd + "/data_csv.csv"
file = pd.read_csv(file_path)
gold_last5years=file.loc[750:,["date","price"]]
result = adfuller(gold_last5years["price"],autolag="AIC")
print ("Results of Dickey-Fuller Test:")
dfoutput = pd.Series(result[0:4], index=['Test Statistic','p-value','#Lags Used','Number of Observations Used'])
for key,value in result[4].items():
    dfoutput['Critical Value (%s)'%key] = value
print (dfoutput)


Results of Dickey-Fuller Test:
Test Statistic                 -2.317467
p-value                         0.166396
#Lags Used                      1.000000
Number of Observations Used    66.000000
Critical Value (1%)            -3.533560
Critical Value (5%)            -2.906444
Critical Value (10%)           -2.590724
dtype: float64


The p-value is obtained 0.167 is greater than significance level of 0.05 and the ADF statistic -2.31 is higher than any of the critical values.

Clearly, there is no reason to reject the null hypothesis. So, the time series is in fact non-stationary.

In [79]:
pwd = os.getcwd()
file_path = pwd + "/silver_price.csv"
file = pd.read_csv(file_path)
silver_last5years=file.loc[12000:,["date","price"]]
result = adfuller(silver_last5years["price"],autolag="AIC")
print ("Results of Dickey-Fuller Test:")
dfoutput = pd.Series(result[0:4], index=['Test Statistic','p-value','#Lags Used','Number of Observations Used'])
for key,value in result[4].items():
    dfoutput['Critical Value (%s)'%key] = value
print (dfoutput)

Results of Dickey-Fuller Test:
Test Statistic                   -1.518910
p-value                           0.524137
#Lags Used                       11.000000
Number of Observations Used    1463.000000
Critical Value (1%)              -3.434828
Critical Value (5%)              -2.863518
Critical Value (10%)             -2.567823
dtype: float64


The p-value is obtained 0.524 is greater than significance level of 0.05 and the ADF statistic -1.52 is higher than any of the critical values.

Clearly, there is no reason to reject the null hypothesis. So, the time series is in fact non-stationary.